app/views/notifications/_reaction.html.erb

```diff
@@ -1,7 +1,2 @@
 <%# missing cache key %>
-<% siblings = notification.json_data["reaction"]["aggregated_siblings"].select { |n| n["created_at"] > 24.hours.ago } %>
-<% if siblings.length == 0 %>
-  <%= render "single_reaction", notification: notification %>
-<% else %>
-  <%= render "aggregated_reactions", notification: notification, siblings: siblings %>
-<% end %>
+<%= notification.inspect %>
```

Comment:
I _think_ I should delete this file, but I could use a second set of :eyes:

I don't think this gets called anymore (because it should always become a `aggregate` or `single`) but I had a hard time convincing myself (it's hard to prove a negative?)